cluster/examples/kubernetes/ceph/read-affinity-config/README.md

```diff
@@ -0,0 +1,14 @@
+# read-affinity-config
+
+This repo holds the scripts for Ceph/RHCS admin to prepare the cluster for implementing read affinity as part of the Metro DR high availability feature in Red Hat OCS 4.7 release.
```

Comment:
The upstream repo shouldn't mention downstream products or reference downstream links. We really need to add upstream documentation on how to use this. But if they are only short term scripts, perhaps we don't need to merge them now and we can wait for the full solution. 

It would be helpful to have a [design doc](https://github.com/rook/rook/tree/master/design/ceph) or at least a github issue for upstream discussion on the approach. Perhaps there are settings in the CephBlockPool CR or other CRs where the operator would be able to automate these actions. Where these scripts are needed in the external cluster, we could follow the pattern we already have for the external cluster handling scripts. For example, see the [external cluster doc](https://rook.github.io/docs/rook/v1.5/ceph-cluster-crd.html#external-cluster).

Reply:
I will answer later today...

Reply:
Upstream users would be able to use Pacific features very soon when it is released in the next month or two, so I assume the "read from non primary" feature would be available then. If these scripts are basically replaced by that feature, seems like we don't need these scripts in upstream Rook. Since the scripts are just for usage downstream, then there is no need to keep them in Rook and no need to come up with upstream documentation either.

Reply:
I found a place for these scripts, I am closing this PR.